src/api/java/com/minecolonies/api/configuration/ConfigurationHandler.java

```diff
@@ -199,7 +209,8 @@ private static int getClampedInt(
                                       final Configuration config, final String category, final String key,
                                       final int defaultValue, final int min, final int max, final String comment)
     {
-        return MathHelper.clamp(config.get(category, key, defaultValue, String.format(FORMAT_RANGE, comment, min, max, defaultValue), min, max).getInt(), min, max);
+        return MathHelper.clamp(config.get(category, key, defaultValue, comment, min, max).getInt(), min, max);
+        //String.format(FORMAT_RANGE, comment, min, max, defaultValue)
```

Comment:
If i remember correctly I fixed this. Without that Format this will break things down the line.!